src/components/MoviesList/MoviesList.js

```diff
@@ -4,15 +4,21 @@ import PropTypes from 'prop-types';
 import './MoviesList.scss';
 import { MovieCard } from '../MovieCard';
 
-export const MoviesList = ({ movies }) => (
+export const MoviesList = ({ movies, query }) => (
   <div className="movies">
-    {movies.map(movie => (
-      <MovieCard key={movie.imdbId} {...movie} />
-    ))}
+    {movies
+      .filter(movie => movie.description
+        .toLowerCase().includes(query.toLowerCase())
+        || movie.title.toLowerCase().includes(query.toLowerCase()))
+      .map(movie => (
```

Comment:
Better to filter the movies on the top level and transfer already prepared and filtered movies to 'MoviesList' components. 
Let's try to separate logic - MoviesList only supposed to render the list of movies